Servers/UI/OJS.Servers.Ui/ClientApp/src/components/portals/administration/AdministrationPortal.tsx

```diff
@@ -534,7 +533,14 @@ const AdministrationPortal = () => {
         <AdministrationThemeProvider mode={currentThemeMode}>
             <CssBaseline />
             <Box>
-                <Box sx={{ display: 'flex', height: '100vh' }}>
+                <Box sx={{
+                    display: 'flex',
+                    overflowY: 'hidden',
+                    '&::-webkit-scrollbar': { display: 'none' },
```

Comment:
What if the user selects more rows per page, will it have scroll then?

Reply:
There will always be a visible scroll if the data cannot be contained on the screen, there were two scrolls at the same time when zooming in, this has been fixed.